terraform/aws/main.tf

```diff
@@ -1,5 +1,5 @@
 resource "template_file" "install" {
-    filename = "${path.module}/scripts/install.sh.tpl"
+    template = "${path.module}/scripts/install.sh.tpl"
```

Comment:
This should use `file()` to pull in the contents of the template. Otherwise we'll get a template with contents that are just this path. :smile: